python/tank/bootstrap/cached_configuration.py

```diff
@@ -88,6 +88,13 @@ def __repr__(self):
             self._descriptor
         )
 
+    @property
+    def descriptor(self):
```

Comment:
what about other `Configuration` subclasses? Seems we should display this in some form on the baseclass to provide a default for all deriving classes?